core/vm/jump_table.go

```diff
@@ -113,7 +113,8 @@ func newNapoliInstructionSet() JumpTable {
 	instructionSet := newShanghaiInstructionSet()
 	enable1153(&instructionSet) // Transient storage opcodes
 	enable5656(&instructionSet) // MCOPY opcode
-	enable6780(&instructionSet) // SELFDESTRUCT only in same transaction
+	// Disable opSelfdestruct6780 for L2
+	//enable6780(&instructionSet) // SELFDESTRUCT only in same transaction
```

Comment:
would it be worth creating a `zkNapoliInstructionSet` function that omits this and comment up the differences there.  Might make taking upstream easier in the future and it will be really explicit